clerk/jwks_cache.go

```diff
@@ -0,0 +1,32 @@
+package clerk
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/square/go-jose.v2"
+)
+
+type jwksCache struct {
```

Comment:
🔧 Since this cache might be called from different goroutines concurrently, it should be made goroutine-safe to prevent data races.